src/pages/index.astro

```diff
@@ -1,38 +1,30 @@
 ---
-import Layout from '../layouts/Layout.astro';
-import Card from '../components/Card/Card.astro';
-import Title from '../components/Title/Title.astro';
-
+import Title from '@components/Title/Title.astro';
+import Hero from '@components/Hero/Hero.astro';
+import Introduction from '@components/Introduction/Introduction.astro';
+import { TITLE_ICONS, TITLE_VARIANTS } from '@domain/constants.ts';
```

Comment:
rozszerzenie `.ts` niepotrzebne

btw, zastanawiam się, czy rozszerzenia `.astro` można by nie podawać — ale to w ramach researchu można sprawdzić przy kolejnym tasku

Reply:
Szukałam już czy można pozbyć się rozszerzenia .astro ale nie znalazłam sposobu. Z tego co zrozumiałam to niestety plików astro nie można importować do innych rodzajów plików np ts/tsx więc importowanie np do index.ts odpada.